perfkitbenchmarker/aws/aws_disk.py

```diff
@@ -33,6 +33,7 @@
 VOLUME_EXISTS_STATUSES = frozenset(['creating', 'available', 'in-use', 'error'])
 VOLUME_DELETED_STATUSES = frozenset(['deleting', 'deleted'])
 VOLUME_KNOWN_STATUSES = VOLUME_EXISTS_STATUSES | VOLUME_DELETED_STATUSES
+DISK_TYPE = {disk.STANDARD: 'standard', disk.SSD: 'gp2', disk.IOPS: 'io1'}
```

Comment:
Should we rename IOPS to PIOPS since this is really not a general type?

I also think we should call SSD REMOTE_SSD to reduce confusion with LOCAL_SSD.